auto-cancel.md

```diff
@@ -0,0 +1,165 @@
+The *auto-cancel* strategy defines the behavior of pipelines when they enter the
```

Comment:
Maybe rephrase this as "when a queue for execution forms"?

As is now it kind of sounds like you're describing any pipeline workflow in daily life...